src/components/modal/__tests__/__snapshots__/CdrModal.spec.js.snap

```diff
@@ -73,7 +73,7 @@ exports[`CdrModal.vue > default open 1`] = `
             <div
               class="cdr-modal__text-content"
               role="document"
-              style="max-height: -80px; padding-right: 0px;"
```

Comment:
@benjag this looks right? We should see a positive value for the opened state correct?

Also, maybe we should have another snapshot for a closed state?

Reply:
Ok, I expect you're right on the nextTick() being too fast then

Reply:
I think this is good to go. I added another snapshot for the closed state.